src/content/docs/security/security-privacy/information-security/software-development-lifecycle.mdx

```diff
@@ -76,18 +76,20 @@ Software built at New Relic goes through these five phases.
+During the design phase, the New Relic security engineer collaborates with the stakeholders, engineering leaders, and architects to get a detailed shared understanding of the feature. Security engineers at New Relic contribute to the design process with stakeholders by creating a threat model documenting any acceptance criteria, features, or requirements to securely implement the feature. Using the threat model, the security engineer adds  detailed specifications for the required controls to the project.
 
 ## Build
-In the build phase, the engineering team implements the features in the project following secure coding standards at New Relic. Each product engineer receives secure coding training, which includes topics such as the [OWASP top 10](https://owasp.org/www-project-top-ten/), input sanitization, and using the secure frameworks and processes already in place at New Relic.
+Each product engineer receives secure coding training which includes topics such as the [OWASP top 10](https://owasp.org/www-project-top-ten/), input sanitization, and using the secure frameworks and process already in place at New Relic. In the build phase the engineering team implements appropriate security features in the project following secure coding standards at New Relic.
 
 ## Verification
-Once feature complete, every pull request must be code reviewed by another engineer with write access to the repository. At the project level, teams may request a final review of the project before deploying. The security engineer verifies that the safeguards and controls discovered and documented in the risk assessment, requirements, and design phases have been addressed.
+Once code complete, every pull request must be code reviewed by another engineer with write access to the code repository. At the project level teams may request a final review of the project before deploying. The security engineer verifies the safeguards and controls discovered and documented in the risk assessment, requirements, and design phase have been addressed.
 
-All New Relic code has passed static code and composition analysis to look for vulnerabilities in the code and dependencies. All files published by New Relic will include their calculated hash and a digital signature. New Relic is in the process of including hashes and signatures, so that anyone downloading files published by New Relic will be able to confirm their downloaded file has not been tampered with and is identical to the one published by New Relic.
+All New Relic code has passed static code and composition analysis to look for vulnerabilities in the code and dependencies. All files published by New Relic will include their calculated hash and a digital signature. New Relic is in the process of including hashes and signatures, anyone downloading files published by New Relic will be able to confirm their downloaded file has not been tampered with and is identical to the one published by New Relic.
```

Comment:
I'd separate these two ideas into separate sentences. How about this:

New Relic is in the process of including hashes and signatures.  Anyone downloading files published by New Relic will be able to confirm their downloaded file has not been tampered with and is identical to the one published by New Relic.

Just to clarify, "Is in the process" means New Relic doesn't do it yet, right? Also, this same comment about published files appears down in Deploy. Is this redundancy necessary?

Reply:
Done